packages/components/package-lock.json

```diff
@@ -14334,13 +14412,14 @@
       }
     },
     "node_modules/react": {
-      "version": "16.14.0",
-      "resolved": "https://registry.npmjs.org/react/-/react-16.14.0.tgz",
-      "integrity": "sha512-0X2CImDkJGApiAlcf0ODKIneSwBPhqJawOa5wCtKbu7ZECrmS26NvtSILynQ66cgkT/RJ4LidJOc3bUESwmU8g==",
+      "version": "16.0.0",
```

Comment:
This doesn't seem right. Switching them to `~` ended up bring them back to the base version.